docs/blogs/EnhancedDevPreviewGatewayAPI/GettingStarted.md

```diff
@@ -0,0 +1,178 @@
+```
+
+Confirm the ingress operator has successfully installed the GatewayClass CRD:
+```shell
+$ oc get crd gatewayclasses.gateway.networking.k8s.io
+NAME                                       CREATED AT
+gatewayclasses.gateway.networking.k8s.io   2023-04-04T18:02:55
+```
+
+Next, create the GatewayClass with the controller name “openshift.io/gateway-controller”. This will instruct the ingress operator to create a SMCP and install the Gateway, HTTPRoute, and Reference Grant CRDs:
+```shell
+$ oc create -f -<<'EOF'
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: GatewayClass
+metadata:
+  name: openshift-default
+spec:
+  controllerName: openshift.io/gateway-controller
+EOF
+```
+
+Confirm the ingress operator has installed the HttpRoute, Gateway, and ReferenceGrant CRDs:
+```shell
+$ oc get crds httproutes.gateway.networking.k8s.io gateways.gateway.networking.k8s.io referencegrants.gateway.networking.k8s.io
+NAME                                        CREATED AT
+httproutes.gateway.networking.k8s.io        2023-04-04T18:02:55Z
+gateways.gateway.networking.k8s.io          2023-04-04T18:02:55Z
+referencegrants.gateway.networking.k8s.io   2023-04-04T18:02:56Z
+```
+
+Confirm the ingress operator has created the SMCP and it is ready:
+```shell
+$ oc get smcp -n openshift-ingress
+NAME                READY   STATUS            PROFILES      VERSION   AGE
+openshift-gateway   5/5     ComponentsReady   ["default"]   2.4.0     107s
+```
+
+OSSM will create two new Istio deployments in the openshift-ingress namespace. Ensure the pods for the deployments are running. These will need to become ready for the SMCP to also go ready:
+```shell
+$ oc get deployment -n openshift-ingress
+NAME                       READY   UP-TO-DATE   AVAILABLE   AGE
+istio-ingressgateway       1/1     1            1           42s
+istiod-openshift-gateway   1/1     1            1           55s
+router-default             2/2     2            2           6h4m
+```
+
+Gateway API and OSSM have been installed successfully and are ready to use. We are now ready to create Gateways and HTTPRoutes.
+
+## Gateway API Example
+
+Let’s now use Gateway API to create a Gateway and Route. In this example, we will convert a simple HTTP OpenShift Route into roughly equivalent Gateway API objects (Gateway, HTTPRoute). This is what the OpenShift Route example would look like:
+```yaml
+apiVersion: v1
+kind: Route
+metadata:
+  name: http
+  namespace: demo-app
+spec:
+  host: demo.gwapi.${DOMAIN}
+  to:
+    kind: Service
+    name: foo-app
+  port:
+    targetPort: 8080
+```
+
+First, create the Gateway with a Listener that will listen for requests on a URL under the cluster domain. The Gateway must be created in the openshift-ingress namespace and specify the GatewayClass created earlier. By default, Istio will automatically provision a gateway deployment and service with the same name upon creation of this Gateway object:
```

Comment:
```suggestion
First, create the Gateway with a Listener (spec.Listeners instance) that will listen for requests on a URL under the cluster domain. The Gateway must be created in the openshift-ingress namespace and specify in spec.gatewayClassName the GatewayClass we created earlier. By default, Istio will automatically provision a gateway deployment and service with the same name upon creation of this Gateway object:
```

Reply:
i would suggest moving the "By default, Istio will automatically provision a gateway deployment and service with the same name upon creation of this Gateway object:" to another section and show the deployment and service.